v2/samples/cache/v1api/v1api20201201_redislinkedserver.yaml

```
@@ -2,15 +2,14 @@ apiVersion: cache.azure.com/v1api20201201
 kind: RedisLinkedServer
 metadata:
   # This name needs to be the same as the secondary redis server.
-  name: sampleredis2-alpha
+  name: linked-sampleredis
```

What's the length limit for these names? If there's room, adding an additional separator would make the easier to read.

```suggestion
  name: linked-sample-redis
```